app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -280,7 +280,7 @@ class GithubWebhookSubscription extends SubscriptionHandler {
     final RepositorySlug slug = pr.base!.repo!.slug();
```

Comment:
Should this instead be one level up in the initial action processor to prevent us from accidentally double scheduling?

```dart
      case 'edited':
        // Editing a PR should not trigger new jobs, but may update whether
        // it has tests.
        await _checkForTests(pullRequestEvent);
        break;
```

This is missing the scheduler call. Based on https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=edited#pull_request, we can check if the ref changed.

I expect something like

```dart
      case 'edited':
        // Editing a PR should not trigger new jobs, but may update whether
        // it has tests.
        await _checkForTests(pullRequestEvent);
        if (pullRequestEvent.changes.base != null) {
          await scheduleIfMergeable(...); 
        }
        break;

Reply:
That is what keyong is saying. I am going to update this accordingly.

Reply:
Shave the yak so to speak. The the call to the scheduler is buried below, which is irrelevant now.

Reply:
I need this to land in order to make this happen much more easily. https://github.com/SpinlockLabs/github.dart/pull/390